docker/docker-stack-compose.yml

```diff
@@ -69,13 +69,11 @@ services:
       make serve
     environment:
       LOG_LEVEL: DEBUG
-      JAVA_OPTS: "-Dplay.filters.hosts.allowed.1=staging.codesearch.aelve.com -Dplay.filters.hosts.allowed.2=cloud.digitalocean.com"
-      # In production you must override the APPLICATION_SECRET key
-      APPLICATION_SECRET: ${SECRET_KEY:-"u?:IPZT:Cun;SP>edigHDs:cpUf2cY\`<7D>b9lHg98^NsVGn]:eLCIjz1Tt_VyIv"}
+      JAVA_OPTS: "-Dplay.filters.hosts.allowed.1=${SERVER_HOST:?}"
```

Comment:
This removes the `cloud.digitalocean.com` host. It's unclear whether we can actually remove it without getting in trouble, though.